Candidate.h

```diff
@@ -0,0 +1,64 @@
+//
+// Created by smark on 11.11.2022.
+//
+#include <string>
+#include <iostream>
+#ifndef LAB5_CANDIDATE_H
+#define LAB5_CANDIDATE_H
+
+namespace C {
+    using namespace std;
+    class Candidate {
+    private:
+        string surname;
+        int votes;
+        double rate;
+    public:
+        Candidate(){
+            surname = "it isn`t set";
+            votes = 0;
+            rate = 404;
+        };
+        Candidate(string a, int b){
+            surname = a;
+            votes = b;
```

Comment:
Please change letters a, b to clearer examples because it doesn't follow the code convention